src/style.css

```diff
@@ -1 +1,72 @@
 /* Styles go here */
+html {
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  display: inline-block;
+  color: inherit;
+  text-decoration: none;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0;
+  padding: 0;
```

Comment:
you can reset the margin and paddin for all the necessary elements at once using a comma-separated enumeration (body, ul { ... }), instead of resetting them for each element individually